docs/content/integrations/dagstermill/using-notebooks-with-dagster.mdx

```diff
@@ -146,47 +146,44 @@ When materialized, the `iris_kmeans_jupyter` asset will execute the notebook (`/
 
 ---
 
-## Step 3: Add a Dagster Repository and supply an I/O manager
+## Step 3: Add a Dagster Definitions object and supply an I/O manager
```

Comment:
There are a lot of places in this doc where i'm not sure if Definitions of code location is the right word to you. @erinkcochran87  what is the standard of when to use Definitions vs code location?

Reply:
This is a tricky one, tbh. This is a TL:DR; of where we landed on this while writing the tutorial:

- A _code location_ is a thing unto itself, and exists even without a `Definitions` object. Code locations are defined by Dagster itself (with or without a `Definitions` object), but when a `Definitions` object *is* defined, the code location will use it to find the definitions it needs to be aware of.

- A `Definitions` object is a way to combine multiple definitions into a single mega-definition, which allows the definitions to be aware of each other. A `Definitions` object isn't analogous to a code location.

All of this to say I think the way we're thinking about this is you can add `Definitions` **to** a code location, but creating a `Definitions` object doesn't create a code location.

I hope that made sense 😅